pages/agent/clusters.md

```diff
@@ -53,6 +53,19 @@ To create additional queues:
 1. Enter a key and description.
 1. Select _Create Queue_.
 
+### Pause a queue
+
+Pausing a queue will prevent jobs from being dispatched to agents associated with that queue. This may affect builds targeting the paused queue.
+
+To pause a queue:
+1. Navigate to your cluster’s _Queues_.
+1. Click on the queue you wish to pause.
+1. Click _Edit_.
+1. Under _Queue Management_, click _Pause Queue_.
+1. Enter an optional note in the dialog if needed, and confirm that you wish to pause the queue.
+
+To resume the queue again, click Resume Queue.
```

Comment:
```suggestion
To resume the queue again, select _Resume Queue_.
```

We're italicising the other bits of UI text here, so this one probably needs it too?

Also, we use "select" instead of "click" as our house style for directly interacting with buttons and other UI elements.